ui/app/serializers/job.js

```diff
@@ -71,6 +71,10 @@ export default class JobSerializer extends ApplicationSerializer {
       .buildURL(modelName, hash.ID, hash, 'findRecord')
       .split('?');
 
+    const variableLookup = hash.ParentID
+      ? JSON.parse(hash.ParentID)[0]
+      : hash.PlainId;
```

Comment:
variableLookup: parameterized and periodic jobs don't have knowable IDs, and therefore our variables won't assign to them directly, but instead to their parents.